interface/main/messages/messages.php

```diff
@@ -23,614 +24,686 @@
-        if($sortby == $sort[$i]) {
-            switch($sortorder) {
-                case "asc"      : $sortlink[$i] = "<a href=\"messages.php?show_all=".attr($showall)."&sortby=".attr($sortby)."&sortorder=desc&$activity_string_html\" onclick=\"top.restoreSession()\"><img src=\"../../../images/sortup.gif\" border=0 alt=\"".htmlspecialchars( xl('Sort Up'), ENT_QUOTES)."\"></a>";
-break;
-                case "desc"     : $sortlink[$i] = "<a href=\"messages.php?show_all=".attr($showall)."&sortby=".attr($sortby)."&sortorder=asc&$activity_string_html\" onclick=\"top.restoreSession()\"><img src=\"../../../images/sortdown.gif\" border=0 alt=\"".htmlspecialchars( xl('Sort Down'), ENT_QUOTES)."\"></a>";
-break;
-            } break;
-        }
-    }
-    // Manage page numbering and display beneath the Messages table.
-    $listnumber = 25;
-    $total = getPnotesByUser($active,$show_all,$_SESSION['authUser'],true);
-    if($begin == "" or $begin == 0) {
-        $begin = 0;
-    }
-    $prev = $begin - $listnumber;
-    $next = $begin + $listnumber;
-    $start = $begin + 1;
-    $end = $listnumber + $start - 1;
-    if($end >= $total) {
-        $end = $total;
-    }
-    if($end < $start) {
-        $start = 0;
-    }
-    if($prev >= 0) {
-        $prevlink = "<a href=\"messages.php?show_all=".attr($showall)."&sortby=".attr($sortby)."&sortorder=".attr($sortorder)."&begin=".attr($prev)."&$activity_string_html\" onclick=\"top.restoreSession()\"><<</a>";
-    }
-    else {
-        $prevlink = "<<";
-    }
+            var SaveNote = function () {
+                <?php if ($noteid) { ?>
+                top.restoreSession();
+                $("#task").val("save");
+                $("#new_note").submit();
+                <?php } ?>
+              }
+
+              var CancelNote = function () {
+                top.restoreSession();
+                $("#task").val("");
+                $("#new_note").submit();
+              }
+          });
+          function gotoReport(doc_id,pname,pid,pubpid,str_dob){
+            EncounterDateArray=new Array;
+            CalendarCategoryArray=new Array;
+            EncounterIdArray=new Array;
+            Count = 0;
+            <?php
+            if(isset($enc_list) && sqlNumRows($enc_list) >0 ){
+                while($row = sqlFetchArray($enc_list)){
+                    ?>
+                  EncounterIdArray[Count]='<?php echo attr($row['encounter']); ?>';
+                EncounterDateArray[Count]='<?php echo attr(oeFormatShortDate(date("Y-m-d", strtotime($row['date'])))); ?>';
+                CalendarCategoryArray[Count]='<?php echo attr(xl_appt_category($row['pc_catname'])); ?>';
+                Count++;
+                <?php
+                }
+            }
+            ?>
+            top.restoreSession();
+            $.ajax({
+              type:'get',
+              url:'<?php echo $GLOBALS['webroot']."/interface/patient_file/encounter/patient_encounter.php";?>',
```

Comment:
This file no longer exists (it was a legacy script that was not used). Created another script to do this:
``````
url:'<?php echo $GLOBALS['webroot']."/library/ajax/set_pt.php";?>',